machine_management/creating-infrastructure-machinesets.adoc

```diff
@@ -10,19 +10,14 @@ include::snippets/machine-user-provisioned-limitations.adoc[leveloffset=+1]
 
 You can use infrastructure machine sets to create machines that host only infrastructure components, such as the default router, the integrated container image registry, and the components for cluster metrics and monitoring. These infrastructure machines are not counted toward the total number of subscriptions that are required to run the environment.
 
-In a production deployment, it is recommended that you deploy at least three machine sets to hold infrastructure components. {SMProductName} deploys Elasticsearch, which requires three instances to be installed on different nodes. Each of these nodes can be deployed to different availability zones for high availability. This configuration requires three different machine sets, one for each availability zone. In global Azure regions that do not have multiple availability zones, you can use availability sets to ensure high availability.
+In global Azure regions that do not have multiple availability zones, you can use availability sets to ensure high availability.
```

Comment:
I defer to whom ever originally wrote this paragraph but I suspect this sentence can be removed as well.  Contextually, it goes along with explaining the other three machine sets while here it is floating tied to nothing.  If anything it may be "attached" to the previous paragraph but likely should go another place where there is information about multiple availability zones.